pkg/record/recorder.go

```diff
@@ -0,0 +1,53 @@
+package record
+
+import "encoding/json"
+
+// OperationKind is the name of a recorded operation
+type OperationKind string
+
+// Params are Operation parameters we can record along with an entry.
+type Params map[string]interface{}
+
+const (
+	// ImportFile is a js import from the filesystem (exclude stdlib imports).
+	ImportFile OperationKind = "import-file"
+	// ReadFile is a std.read from the filesystem (exclude reading from stdin).
+	ReadFile OperationKind = "read-file"
```

Comment:
What about params file? Or would that not quite work

Reply:
That's a great idea! will add them.